doc/Makefile

```diff
@@ -1,8 +1,13 @@
 ### Makefile - for DOCUMENTATION (./doc) of ESS distribution.
 ###
 
-## Before making changes here, please take a look at Makeconf
 include ../Makeconf
+LOCAL_CONF := $(strip $(wildcard ../Makeconf.local))
```

Comment:
I guess, you could add this at the end of `Makeconf` and not alter all these makefiles.